README.md

```diff
@@ -1,50 +1,107 @@
-Updated based on above debug commands and current ID template for the README.
-@ericabr please review for any missing content/improvements:
-
 # ibm-commonui-operator
 
-You can use the ibm-commonui-operator to install the Common Web UI service for the IBM Cloud Platform Common Services and access the Common Web UI console.
+> **Important:** Do not install this operator directly. Only install this operator using the IBM Common Services Operator. For more information about installing this operator and other Common Services operators, see [Installer documentation](http://ibm.biz/cpcs_opinstall).
+> If you are using this operator as part of an IBM Cloud Pak, see the documentation for that IBM Cloud Pak to learn more about how to install and use the operator service. For more information about IBM Cloud Paks, see [IBM Cloud Paks that use Common Services](http://ibm.biz/cpcs_cloudpaks).
```

Comment:
A line break should be added between the 2 paragraphs in the "Important note"

Reply:
Goes against the linter, can't do it